lib/components/Toastr.js

```diff
@@ -26,17 +26,21 @@ const ToastrComponent = ({ type, message }) => {
   );
 };
 
-const showToastr = message => {
+const showToastr = (message) => {
   toast.success(<ToastrComponent type="success" message={message} />, {
     position: toast.POSITION.BOTTOM_CENTER,
     transition: Slide,
   });
 };
 
-const isError = e => e && e.stack && e.message;
+const isError = (e) => e && e.stack && e.message;
 
-const showErrorToastr = error => {
-  const errorMessage = isError(error) ? error.message : error;
+const showErrorToastr = (errorObject) => {
+  let errorMessage;
+  if (isError(errorObject)) errorMessage = errorObject.message;
+  else if (errorObject.error) errorMessage = errorObject.error;
+  else if (errorObject.errors) errorMessage = errorObject.errors[0];
```

Comment:
@josephv7 _a
Lets join the errors in the array into string
eg : R.join("\n", errorObject.errors);  ( Ref  neetoHelp )